core/bsdb/bsdb.go

```diff
@@ -0,0 +1,6 @@
+package bsdb
```

Comment:
should add ReadMe.md for what's bsdb and metadata?

Reply:
added later in https://github.com/bnb-chain/greenfield-storage-provider/pull/471